### PR TITLE
HAITI-C: Created Common csv file

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

File: configuration/concepts/common.csv [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+144dd787-a13f-437d-b6c7-9a7d4cf3bc56,,CIEL:162398,Dose,Dose,Dose,Dose,,,Misc,N/A,,,,,,,,,,,
+f36097e7-7d66-4196-8d95-6b66f31435cc,,,See comments,Commentaires,See comments,Commentaires,,,Frequency,N/A,,,,,,,,,,,
+9d5b708e-3f10-11e4-adec-0800271c1b75,,CIEL:162384,Dosing Units,Unités de dosage,dosing units,Unités de dosage,,,Misc,N/A,,,Capsule(s); Tablet(s); ml; mg; IU; Drop; Tablespoon; Teaspoon; Unit(s); Puff(s); Application; Packet; Dose; See comments,,,,,,,,
+0de96a2d-5c2c-4110-9770-914369de1713,,,Opthalmic,Ophtalmique,Opthalmic,Ophtalmique,,,Misc,N/A,,,,,,,,,,,
+99cbeb1b-621c-4645-a600-d702364d9885,,,Otic,Otique,Otic,Otique,,,Misc,N/A,,,,,,,,,,,
+9d6c9153-3f10-11e4-adec-0800271c1b75,,CIEL:162392,Intravaginal,Intravaginale,Intravaginal,Intravaginale,,,Misc,N/A,,,,,,,,,,,
+9d697972-3f10-11e4-adec-0800271c1b75,,CIEL:162394,Drug Routes,Voie d'administration,Drug Routes,Voie d'administration,,,Misc,N/A,,,Oral; Topical; Inhalation; Intramuscular; Intravenous; Intravaginal; Opthalmic; Otic; See comments;,,,,,,,,
+9d77abd7-3f10-11e4-adec-0800271c1b75,,CIEL:160862,Daily,du quotidien,OD,OD,,,Frequency,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/common.csv]: in fr: "Quotidien"

File: configuration/concepts/common.csv [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+335071ea-8a92-11e4-977f-0800271c1b75,,,Q48H,Q48H,Q48H,Q48H,,,Frequency,N/A,,,,,,,,,,,
+33516e22-8a92-11e4-977f-0800271c1b75,,CIEL:1099,Weekly,Hebdomadaire,Weekly,Hebdomadaire,,,Frequency,N/A,,,,,,,,,,,
+9d8e6918-3f10-11e4-adec-0800271c1b75,,CIEL:161235,Now,Maintenant,STAT,STAT,,,Frequency,N/A,,,,,,,,,,,
+2a148983-c143-4649-b4be-f8845c934ec0,,,Nightly,Chaque nuit,Nightly,Chaque nuit,,,Frequency,N/A,,,,,,,,,,,
+,,CIEL:1065,Yes,Oui,Yes,Oui,,,Misc,N/A,,,,,,,,,,,
+,,CIEL:1066,No,Non,No,Non,,,Misc,N/A,,,,,,,,,,,
+26c0366b-83e4-4401-9699-e545219b8870,,CIEL:5622,Other,Autre,Other,Autre,,,Misc,N/A,,,,,,,,,,,
+75e366a1-c441-4c94-9c53-312c9b8cc550,,CIEL:1175,Not Applicable,Non applicable,N/A,N/A,,,Misc,N/A,,,,,,,,,,,
+6b32bbbe-8a7c-4160-99ff-0dd6b255f458,,CIEL:1067,Unknown,Inconnu,Unknown,Inconnu,,,Misc,N/A,,,,,,,,,,,
+036f30c9-ccff-4752-8206-722724cbbf49,,CIEL:1115,Normal,Ordinaire,Normal,Ordinaire,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/common.csv]: in fr "Normal"

File: configuration/concepts/common.csv [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+7c4b5dff-ade6-4fab-9452-b3be4bcba89e,,CIEL:1116,Abnormal,Anormal,Abnormal,Anormal,,,Misc,N/A,,,,,,,,,,,
+f7139b53-6e10-4096-8e87-ea133fc8fc46,,CIEL:164371,Not Seen,Non vu,Not Seen,Non vu,,,Misc,N/A,,,,,,,,,,,
+48be742e-e22e-4bd9-bb2b-0d4840c60aeb,,,Boy,Garçon,Boy,Garçon,,,Misc,N/A,,,,,,,,,,,
+7adb833e-9e48-4b76-aa8f-24daa3acb810,,,Girl,Fille,Girl,Fille,,,Misc,N/A,,,,,,,,,,,
+5c75e55f-845c-4df6-a6da-b45475a4a215,,,Weight Percentile,Percentile,Weight Percentile,Percentile,,,Misc,Numeric,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/common.csv]: I am surprised there isn't a CIEL mapping for such a generic concept.

File: configuration/concepts/common.csv [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+d291cf12-a568-4eec-9f86-9ef55333c3b2,,,Risky Pregnancy,Grossesse à risque,Risky Pregnancy,Grossesse à risque,,,Misc,Coded,,Yes; No,,,,,,,,,
+ee39e8af-12e8-41ae-a1f9-b08acea07587,,,Other Pregnancy Risks,Autres risques de grossesse,Other Pregnancy Risks,Autres risques de grossesse,,,Misc,Text,,,,,,,,,,,
+ca69545b-dfef-4210-b311-dc8a1d309311,,CIEL:540,Anterior,Antérieur,Anterior,Antérieur,,,Misc,N/A,,,,,,,,,,,
+1f82cf3c-ce84-4c98-a35a-c37c45edc5fc,,CIEL:541,Posterior,Postérieur,Posterior,Postérieur,,,Misc,N/A,,,,,,,,,,,
+f5c721f6-7162-4533-9a83-4a4cd6997cb8,,CIEL:542,Lateral,Lateral,Lateral,Lateral,,,Misc,N/A,,,,,,,,,,,
+4d1f668f-f9b7-4e95-8436-53f37e8c643e,,,Other observations,Autres observations,Other observations,Autres observations,,,Misc,Text,,,,,,,,,,,
+b0823b2c-4bc8-2a7c-41b5-7bf3c7f14e82,,,Duration,Durée,Duration,Durée,,,Misc,Numeric,,,,,,,,,,,
+4f6bc54d-6168-881b-86c7-7c4f1d7a4e9e,,CIEL:1074,Months,Mois,Months,Mois,,,Misc,N/A,,,,,,,,,,,
+a4d0c480-923b-49fe-b125-80453675fdea,,CIEL:1734,Years,An(s),Years,An(s),,,Misc,N/A,,,,,,,,,,,
+850c5ae8-38eb-270b-2ff2-e4ade2e95faf,,CIEL:1822,Hours,Heures,Hours,Heures,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/common.csv]: in fr:
Heure(s)
Jour(s)
Semaine(s)

in en:
Month(s)
Year(s)
Hour(s)
Day(s)
Week(s)

File: configuration/concepts/common.csv [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+327cdb97-571c-2d13-306b-4024fce268cb,,CIEL:1072,Days,Jours,Days,Jours,,,Misc,N/A,,,,,,,,,,,
+1c4d0d14-8ea9-7755-3ae7-03c88e66a41a,,CIEL:1073,Weeks,Semaines,Weeks,Semaines,,,Misc,N/A,,,,,,,,,,,
+ae703c82-ed81-4b14-be4b-966949a1a696,,,Acute,Aigue,Acute,Aigue,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/common.csv]: Let's use singluar: "Aigu"

File: configuration/concepts/common.csv [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+9de275a5-d92c-4868-8430-973ebcc9d593,,,Chronic,Chronique,Chronic,Chronique,,,Misc,N/A,,,,,,,,,,,
+373b5749-82ff-a3d8-2e13-db76dd0499b3,,,Time Units,Unités de temps,Time Units,Unités de temps,,,Misc,Coded,,Hours;Days;Weeks;Months;Years,,,,,,,,,
+c85623b5-7630-246e-a77e-65b146c713f1,,,Acute/Chronic,Aigue/Chronique,Acute/Chronic,Aigue/Chronique,,,Misc,Coded,,Acute;Chronic,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/common.csv]: "Aigu/Chronique"

**Review by rbuisson — CHANGES_REQUESTED** [review]
Just few translation changes and support of _singular(plural)_ in units of time.

File: configuration/concepts/common.csv (hunk 3 above)
Comment (reagan-meant) [reply configuration/concepts/common.csv]: An oversight on my end..Added

File: configuration/concepts/common.csv (hunk 1 above)
Comment (reagan-meant) [reply configuration/concepts/common.csv]: Done

File: configuration/concepts/common.csv (hunk 2 above)
Comment (reagan-meant) [reply configuration/concepts/common.csv]: Done

File: configuration/concepts/common.csv (hunk 4 above)
Comment (reagan-meant) [reply configuration/concepts/common.csv]: Done

File: configuration/concepts/common.csv (hunk 5 above)
Comment (reagan-meant) [reply configuration/concepts/common.csv]: Done

File: configuration/concepts/common.csv (hunk 6 above)
Comment (reagan-meant) [reply configuration/concepts/common.csv]: Done

**Pull request merged**